binds/python/bind_immutable.cpp

```diff
@@ -53,6 +53,10 @@ static void bind_immutable_module(py::module &m) {
                 return py::array(static_cast<py::ssize_t>(morpho->points().size()), morpho->points().data());
             },
             "Returns a list with all points from all sections")
+        .def_property_readonly("section_offsets", [](morphio::Morphology* morpho){
+                return py::array(static_cast<py::ssize_t>(morpho->sectionOffsets().size()), morpho->sectionOffsets().data());
```

Comment:
why the static cast to ssize_t when the type is uin32_t?

Reply:
If you don't you get:
>  /home/bcoste/workspace/io/binds/python/bind_immutable.cpp:53:55: error: conversion to ‘pybind11::ssize_t {aka long int}’ from ‘std::vector<std::array<float, 3> >::size_type {aka long unsigned int}’ may change the sign of the result [-Werror=sign-conversion]